src/sentry/release_health/metrics.py

```diff
@@ -359,6 +362,15 @@ def _count_users(total: bool, referrer: str) -> Dict[Any, int]:
 
         return rv
 
+    def run_sessions_query(
+        self,
+        org_id: int,
+        query: QueryDefinition,
+        span_op: str,
+    ) -> SessionsQueryResult:
+
+        return run_sessions_query(org_id, query, span_op)
```

Comment:
I'd rename to leading underscore to ensure we don't have stray usages

Reply:
Did that for the original module now:
https://github.com/getsentry/sentry/blob/cd8e4a2184d65b810c83b090ca7e6d5c297aa063/src/sentry/snuba/sessions_v2.py#L388
This instance is a new function from a new module though.